src/components/Instafeed/Instafeed.scss

```diff
@@ -0,0 +1,19 @@
+@import "styles/colors.scss";
+@import "styles/mixins.scss";
+
+.instafeed {
+  h3 {
+    @include dynamic-text(2.5em);
+    font-weight: 350;
+  }
+  .crt-logo {
+    color: white !important;
+  }
+  .crt-play-icon {
+    display: none;
```

Comment:
Can be done with tailwind.

Reply:
This is part of the curator library that I don't have the ability to style, I would prefer to leave it in css.

Reply:
No problemo.